mne/tests/test_event.py

```diff
@@ -203,14 +203,31 @@ def test_find_events():
     # 1 == '0b1', 2 == '0b10', 3 == '0b11', 4 == '0b100'
 
     assert_raises(TypeError, find_events, raw, mask="0")
-    assert_array_equal(find_events(raw, shortest_event=1, mask=1),
+    assert_array_equal(find_events(raw, shortest_event=1, mask=1,
+                       mask_type='not_and'),
```

Comment:
I wouldn't change the tests here and below, but add one test to ensure that the default is 'not_and'.